Epona/include/Epona/Analysis.hpp

```diff
@@ -158,71 +168,126 @@ void FindExtremalVertices(
 }
 
 /**
- *  @brief  Calculates box vertices in the world coordinate space from a given
- *          orthogonal basis and it's position
+ * @brief Calculates vertex position in the world space
  *
- *  Writes output vertices to the container starting with @p verticesBeginIterator
+ * @tparam  Vector the vector type that has overloaded vector-vector addition operator
+ * @tparam  Matrix the matrix type that has overloaded matrix-vector multiplication operator
  *
- *  @attention  There must be at least 7 more elements following given iterator
+ * @param   vertex      the vertex data
+ * @param   translation the vertex translation vector
+ * @param   rotation    the vertex rotation matrix
```

Comment:
if `translation` and `rotation` have nothing to do with vertex when passed to this method, don't mention vertex in their description